tests/test_branch_command.rs

```diff
@@ -526,7 +526,7 @@ fn test_branch_list_filtered_by_revset() {
     "###);
 
     // Can't select deleted branch.
-    insta::assert_snapshot!(query("remote-delete"), @r###"
+    insta::assert_snapshot!(query("branches(remote-delete)"), @r###"
```

Comment:
Optional: add a test showing the old call errors out? I guess you could make a `query_error` helper function.

Reply:
Make sense. Add test.